queries/leo/highlights.scm

```diff
@@ -0,0 +1,264 @@
+((identifier) @variable (#set! "priority" 95))
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
+ "block"
+ "console"
+ "const"
+ "constant"
+ "finalize"
+ "in"
+ "let"
+ "mapping"
+ "private"
+ "program"
+ "public"
+ "record"
+ "self"
+ "struct"
+] @keyword
+
+[
+ "transition"
+ "function"
+ "inline"
+] @keyword.function
+
+"import" @include
+
+"return" @keyword.return
+
+"for" @repeat
+
+[ 
+  "if"
+  "then"
+  "else"
+] @conditional
+
+[ ";" "," "::"] @punctuation.delimiter
+
+[
+"!"
+
+"&&"
+"||"
+
+"=="
+"!="
+
+"<"
+"<="
+">"
+">="
+
+"&"
+"|"
+"^"
+
+"<<"
+">>"
+
+"+"
+"-"
+"*"
+"/"
+"%"
+"**"
+
+"="
+
+"+="
+"-="
+"*="
+"/="
+"%="
+"**="
+
+"<<="
+">>="
+
+"&="
+"|="
+"^="
+
+"&&="
+"||="
+
+] @operator
+
+(comment) @comment
+
+(boolean_literal) @boolean
+
+(ternary_expression
+  (_)
+  (ternary_if) @conditional
+  (_)
+  (ternary_else) @conditional
+  (_))
+
+(constant_declaration 
+  (identifier) @constant
+)
+
+;program name if with right extension "aleo" -> okay
+(program_declaration
+  (program_id 
+    name: (identifier) 
+    extension: (identifier) @ext (#eq? @ext "aleo")) @string
```

Comment:
invalid capture name, change it to `@_ext`

Reply:
fixed: [0fb5f122](https://github.com/r001/nvim-treesitter/commit/0fb5f122fb285def4d507e765380bc4f01f7e278)